recipe/recipe.yaml

```
@@ -1,39 +1,39 @@
-{% set name = "bash" %}
-{% set version = "5.2.21" %}
+context:
+  name: bash
+  version: "5.2.37"
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  - url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: c8e31bdc59b69aaffc5b36509905ba3e5cbb12747091d27b4b977f078560d5b8
+  - url: https://ftp.gnu.org/gnu/${{ name }}/${{ name }}-${{ version }}.tar.gz
+    sha256: 9599b22ecd1d5787ad7d3b7bf0c59f312b3396d1e281175dd1f8a4014da621ff
 
 build:
-  number: 1
-  skip: True  # [win]
+  number: 0
+  skip: win
 
 requirements:
   build:
     - gnuconfig  # [unix]
-    - {{ compiler('c') }}
-    - {{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
     - make
     - bison
   host:
     - readline
   run:
     - readline
 
-test:
-  commands:
-    - bash --version | grep {{ version }}
+tests:
+  - script:
+    - bash --version | grep 5.2
```

Could this be
```suggestion
    - bash --version | grep ${{ version }}
```

instead?

---

That doesn't yet work in v1 recipes sadly, because we defer the script rendering and don't have the variable available anymore. It'll be fixed at some point.